src/SFML/Window/WindowImpl.cpp

```diff
@@ -199,6 +198,8 @@ void WindowImpl::processJoystickEvents()
 
         if (connected)
```

Comment:
This is an example of a good change, as it could (depending on platform) save an expensive joystick query. It probably doesn't have huge weight compared to other joystick/input operations, but it also _increases_ code clarity as it moves declarations more local, so win/win.

The only thing I hope is that this call did not on one platform have a side effect that would be required 😬 but in that case it would be a hidden bug, so don't worry.

Reply:
@Bromeon fix it.